assets/js/src/core/components/tree/components/context-menu/context-menu.tsx

```diff
@@ -94,7 +93,7 @@ export const TreeContextMenu = ({ children, node }: TreeContextMenuProps): React
 
           if (formData.has('zipFile')) {
             // @ts-expect-error - marvin will know why
-            await dispatch(archiveUpload.initiate({ body: formData, parentId: parseInt(node.id) })).unwrap()
+            await dispatch(archiveUpload.initiate({ body: formData, parentId: parseInt(node.id) }))
```

Comment:
Same here, let's move it to modules, somehow. :)

Reply:
done with https://github.com/pimcore/studio-ui-bundle/pull/527/commits/7ac29efdc39882e0150aa4f8cc710aaa6b66fc4c